### DISCUSSION TITLE
is it the code , the model or the memory ?

### DISCUSSION THREAD

**Discussion by m0st4f4-hub** [rhohndorf/Auto-Llama-cpp#13] [Q&A]
I'm confused guys , I've been stuck at this point for a while

C:\AutoLLma\Auto-Llama-cpp>python scripts\main.py
llama.cpp: loading model from ./models/ggml-vicuna-13b-4bit.bin
llama_model_load_internal: format     = ggjt v1 (latest)
llama_model_load_internal: n_vocab    = 32001
llama_model_load_internal: n_ctx      = 2048
llama_model_load_internal: n_embd     = 5120
llama_model_load_internal: n_mult     = 256
llama_model_load_internal: n_head     = 40
llama_model_load_internal: n_layer    = 40
llama_model_load_internal: n_rot      = 128
llama_model_load_internal: ftype      = 2 (mostly Q4_0)
llama_model_load_internal: n_ff       = 13824
llama_model_load_internal: n_parts    = 1
llama_model_load_internal: model size = 13B
llama_model_load_internal: ggml ctx size =  73.73 KB
llama_model_load_internal: mem required  = 9807.47 MB (+ 1608.00 MB per state)
llama_init_from_file: kv self size  = 1600.00 MB
AVX = 1 | AVX2 = 1 | AVX512 = 0 | FMA = 1 | NEON = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 0 | SSE3 = 1 | VSX = 0 |
Welcome back!  Would you like me to return to being Entrepreneur-GPT?
Continue with the last settings?
Name:  Entrepreneur-GPT
Role:  an AI designed to autonomously develop and run businesses with the sole goal of increasing your net worth.
Goals: ['Increase net worth.', 'Develop and manage multiple businesses autonomously.', 'Play to your strengths as a Large Language Model.']
Continue (y/n): y
Using memory of type: LocalCache
| Thinking...
llama_print_timings:        load time =  3347.00 ms
llama_print_timings:      sample time =     0.00 ms /     1 runs   (    0.00 ms per run)
llama_print_timings: prompt eval time =  3346.81 ms /     2 tokens ( 1673.41 ms per token)
llama_print_timings:        eval time =     0.00 ms /     1 runs   (    0.00 ms per run)
llama_print_timings:       total time =  3347.41 ms
**Warning: model not found. Using cl100k_base encoding.
Warning: model not found. Using cl100k_base encoding.**
| Thinking...

**Comment by m0st4f4-hub** [comment]
it's memory indeed , trying to switch to a smaller model that can live on my 10gb left ram , any hints about using gpt4all ?

**Reply by Vitorhsantos** [reply]
I manage to use GPT4ALL (the old version, since the new one GPT4ALL-J doesn't work with llama.cpp yet) but don't waste your time, vicuna is still the best model.

**Reply by m0st4f4-hub** [reply]
I appreciate the tip

**Comment by m0st4f4-hub** [comment]
Been trying miserably to switch to gpt4all as it worked stand alone on my device  , but still no luck .On Apr 18, 2023 8:32 AM, Vitorhsantos ***@***.***> wrote:
I have the same warning everytime, did you figured out how to execute without warnings?
